### PR TITLE
Add SaveAll command

### PR DESCRIPTION
**Pull request by Kvaz1r**
Add SaveAll command. 

Unfortunately I could not do save through the call function Save for the current editor.

### CODE REVIEW

File: editor/tabbed_editor.go [hunk 1]
```diff
@@ -139,6 +141,20 @@ func (e *TabbedEditor) CloseCurrentEditor() (name string, editor *CodeEditor) {
...
+		if err != nil {
+			log.Printf("Could not save %s : %s", name, err)
+		}
+		defer f.Close()
+		if _, err := f.WriteString(editor.Text()); err != nil {
+			log.Printf("Could not write to file %s: %s", name, err)
+		}
+	}
+}
+
```
Comment (Kvaz1r) [inline editor/tabbed_editor.go]: yes, I thought about this, but not found way for easy traversal of inner editors.

Comment (nelsam) [inline editor/tabbed_editor.go]: We can add a `SaveAll()` method to the `MultiEditor` interface, [here](https://github.com/nelsam/vidar/blob/master/editor/split_editor.go#L41-L51).  That would force both the `TabbedEditor` and `SplitEditor` to implement `SaveAll()`, which is really what we want.

Comment (Kvaz1r) [inline editor/tabbed_editor.go]: thanks, will do so.

File: commands/save_all.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+func (s *SaveAll) Name() string {
+	return "save-all-files"
+}
+
+func (s *SaveAll) Menu() string {
+	return "File"
+}
+
+func (s *SaveAll) Exec(target interface{}) (executed, consume bool) {
+	saver, ok := target.(EditorWithSave)
```
Comment (nelsam) [inline commands/save_all.go]: We're still type asserting to `EditorWithSave`, here.  Does this compile?  I think it needs to assert to `AllSaver`, right?

**Pull request merged**